### ISSUE TITLE
throws error

### ISSUE THREAD

**Issue by burningTyger** [fraserxu/electron-pdf#6]
I've just installed and probably I'm doing it wrong:

```shell
~ electron-pdf http://localhost:9393/some/doc test.pdf
App threw an error when running [SyntaxError: Unexpected token =]
A JavaScript error occurred in the main process
Uncaught Exception:
SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/me/.nvm/versions/node/v4.2.1/lib/node_modules/electron-pdf/index.js:7:13)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
```

Any idea what's going on here?

**Comment by fraserxu** [comment]
Can you give me more information about your system and which version are you using?

Also I see you are using node v4.2.1 here, when you install the module are you using the same version?

If so you may try to reinstall it.

**Comment by burningTyger** [comment]
I'm on Arch Linux. I used nvm to install node v4.2.1 then installed as is recommended via npm. I can run electron though.

**Comment by JStumpp** [comment]
Same problem here, on node v4.2.1 on windows.

**Comment by fraserxu** [comment]
Ok, I'll try to find a windows machine tomorrow and test it out.

**Comment by justsml** [comment]
I'm on Debian w/  Kernel v3.16

```sh
# uname -a
Linux dev30 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt11-1+deb8u3 (2015-08-04) x86_64 GNU/Linux
```
Node is v0.12.4
NPM v2.13.5


Here's the stacktrace 

```sh
# electron-pdf
App threw an error when running [SyntaxError: Unexpected token =]
A JavaScript error occurred in the main process
Uncaught Exception:
SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/nvm/versions/node/v0.12.4/lib/node_modules/electron-pdf/index.js:7:13)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)

```

**Comment by fraserxu** [comment]
:sweat_smile: Too much ES6... Thanks @JStumpp

**Comment by fraserxu** [comment]
Merged and pushed in v0.4.2, closing...

**Comment by burningTyger** [comment]
It starts again, thank you for fixing. However, I can't produce a PDF now. I'm using your README example.
